cdap-ui/app/features/adapters/controllers/tabs/runs/tabs/status-ctrl.js

```diff
@@ -57,4 +59,61 @@ angular.module(PKG.name + '.feature.adapters')
 
 
       });
+
+    var context = 'namespace.' + $state.params.namespace +
```

Comment:
I suggest using tags, instead of context. So it'll be a dictionary rather than a string. This avoids needing the call to `MyMetricsQueryHelper.contextToTags`.
```
var tags = { namespace: $state.params.namespace,
             adapter: $state.params.adapterId };
var tagQueryParams = MyMetricsQueryHelper.tagsToParams(tags);
```